### DISCUSSION TITLE
Material Height Limit?

### DISCUSSION THREAD

**Discussion by KatKiing** [scfmod/FS22_TerraFarm#119] [Q&A]
![Screenshot 2024-12-09 221127](https://github.com/user-attachments/assets/a40231ee-db04-4468-855a-402cb8bf6fa8)
it will not let me drop material on my pile past a certain point. Is this normal, or is there a way to get around it? Is it map specific?
Thank you, sorry if this question has been asked too much

**Comment by scfmod** [comment]
It's defined by the map
